src/main/java/com/epam/aidial/core/service/InvitationService.java

```diff
@@ -125,6 +126,30 @@ public InvitationCollection getMyInvitations(String bucket, String location) {
         return new InvitationCollection(new HashSet<>(invitationMap.getInvitations().values()));
     }
 
+    public void cleanUpResourceLinks(String bucket, String location, Set<ResourceLink> resourcesToCleanUp) {
+        ResourceDescription resource = ResourceDescription.fromDecoded(ResourceType.INVITATION, bucket, location, INVITATION_RESOURCE_FILENAME);
+        resourceService.computeResource(resource, state -> {
+            InvitationsMap invitations = ProxyUtil.convertToObject(state, InvitationsMap.class);
+            if (invitations == null) {
+                invitations = new InvitationsMap(new HashMap<>());
```

Comment:
I think we don't need to create an empty map and proceed the request further.
Just make a return here.